woocommerce/woocommerce
# Remove unused php imports from BlockTemplatesRegistry that were accidentally left in a previous commit

### Changes proposed in this Pull Request:

This removes 2 unused imports. The usage of them was removed in ddabdb1bc2b0c29f53008ce7418f6a36bbcf5252 but the import was not.


### How to test the changes in this Pull Request:

See CI pass, you could also sanity check on trunk to see that the imports are unused.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 
Removal of an unused import is a non-functional change.

</details>
